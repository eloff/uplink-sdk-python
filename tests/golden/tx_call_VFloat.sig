r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 34957863381158273449956659282646884489679292291101425560454705702374524781644
