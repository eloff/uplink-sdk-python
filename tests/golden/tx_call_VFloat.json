{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": 3.5, 
                        "tag": "VFloat"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zNDk1Nzg2MzM4MTE1ODI3MzQ0OTk1NjY1OTI4MjY0Njg4NDQ4OTY3OTI5MjI5MTEwMTQyNTU2MDQ1NDcwNTcwMjM3NDUyNDc4MTY0NA=="
}